### PR TITLE
Issue 174: Add FAQ website section

### LINKED ISSUE #174: Create FAQ or tips section
As a part of issue #163 there was discussion about whether we should provide functionality for "fitting models safely".

I think a nice solution to something like this could be to provide an FAQ section somewhere on the website (perhaps a vignette) where we answer common questions about the package, provide suggested workflows to do more advanced things, suggest other packages which are useful.

For this issue, I'd suggest to:

* [ ] Determine where this FAQ section could be hosted. Natural option IMO is as a vignette
* [ ] Write the FAQ section for using `purrr` for safely running many models. Maybe also use of `map`

### PR DESCRIPTION
**Pull request by athowes**
## Description

This draft PR will close #174.

## Checklist

- [ ] My PR is based on a package issue and I have explicitly linked it.
- [ ] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [ ] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [ ] I have tested my changes locally.
- [ ] I have added or updated unit tests where necessary.
- [ ] I have updated the documentation if required.
- [ ] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [ ] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

**Comment by athowes** [comment]
This can be reviewed (modulo test passes).

I am having trouble checking it works locally because I get this error running `pkgdown::build_site()`:

```
> pkgdown::build_site()
── Installing package epidist into temporary library ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
── Building pkgdown site for package epidist ───────────────────────────────────
Reading from: /Users/adamhowes/Documents/cfa/epidist
Writing to: /Users/adamhowes/Documents/cfa/epidist/docs
── Sitrep ──────────────────────────────────────────────────────────────────────
✔ URLs ok.
✔ Favicons ok.
✔ Open graph metadata ok.
✔ Articles metadata ok.
✔ Reference metadata ok.
── Initialising site ───────────────────────────────────────────────────────────
── Building home ───────────────────────────────────────────────────────────────
Reading LICENSE.md
Error: 
! in callr subprocess.
Caused by error in `readRDS(path)`:
! error reading from connection
ℹ See `$stdout` and `$stderr` for standard output and error.
Type .Last.error to see the more details.
```

File: vignettes/faq.Rmd [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{FAQ}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+---
+
+```{r include=FALSE}
+library(epidist)
```
Comment (seabbs) [inline vignettes/faq.Rmd:20]: if you hide thsi how are peope reading it going to reproduce? What are you thinking the workflow will look like here?

**Review by seabbs — APPROVED** [review]
This looks really good. One question about the proposed workflow for users

File: vignettes/faq.Rmd (hunk 1 above)
Comment (athowes) [reply vignettes/faq.Rmd:20]: Yeah good point / something I'd thought a little about but think there are better solutions. It's pretty typical that in docs like this they aren't reproducible. But I think it should be reproducible. Maybe have it in a > section (code folded). Don't want to distract from the main body. What do you think?

**Comment by athowes** [comment]
Before merge I think we should add a section saying how we want people to ask questions. How do we want? Issues? Something different?

Also we should add an issue to brainstorm more tips / FAQ.

**Comment by athowes** [comment]
Also anyone have any idea what might be going on in my error (https://github.com/epinowcast/epidist/pull/181#issuecomment-2238814762) above?

**Comment by athowes** [comment]
Added issue https://github.com/epinowcast/epidist/issues/182. Ready for re-review

File: vignettes/faq.Rmd (hunk 1 above)
Comment (seabbs) [reply vignettes/faq.Rmd:26]: I like the code folding approach. Nice one

**Comment by seabbs** [comment]
No no idea. As it isn't happening in the CI I guess you have a different (and likely problematic) version of something vs what the CI sees

**Review by seabbs — DISMISSED** [review]

**Comment by seabbs** [comment]
I'm a bit confused why this runs with `bayesplot` being present given we don't have it as a suggests: https://github.com/epinowcast/epidist/blob/55ab8bf12517dfb29d09f7251e8c25f827a6e3c7/DESCRIPTION#L46

**Comment by athowes** [comment]
> I'm a bit confused

Hmm yeah good point. Is it that it's imported by something?

**Comment by athowes** [comment]
> No no idea. As it isn't happening in the CI I guess you have a different (and likely problematic) version of something vs what the CI sees

Yeah it was a bad version of `httr2` I found.

**Comment by athowes** [comment]
Yeah `brms` imports `bayesplot`

**Review by seabbs — APPROVED** [review]

**Pull request merged**